pkg/golinters/gomnd.go

```diff
@@ -8,20 +8,29 @@ import (
 	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
 )
 
-func NewGoMND(cfg *config.Config) *goanalysis.Linter {
-	analyzers := []*analysis.Analyzer{
-		mnd.Analyzer,
-	}
-
+func NewGoMND(settings *config.GoMndSettings) *goanalysis.Linter {
 	var linterCfg map[string]map[string]interface{}
-	if cfg != nil {
-		linterCfg = cfg.LintersSettings.Gomnd.Settings
+
+	if settings != nil {
+		// TODO(ldez) For compatibility only, must be drop in v2.
+		if len(settings.Settings) > 0 {
+			linterCfg = settings.Settings
+		} else {
+			linterCfg = map[string]map[string]interface{}{
+				"mnd": {
```

Comment:
did you mean `gomnd` ?

Reply:
The analyzer name is `mnd`, so it's the right value.

https://github.com/tommy-muehle/go-mnd/blob/c5e4f67f0b1b0a8114d42c8b95709f4e6dd30b37/analyzer.go#L19

Reply:
I see error too. You are right.